libs/core/langchain_core/runnables/graph_mermaid.py

```diff
@@ -306,6 +306,7 @@ def _render_mermaid_using_api(
     mermaid_syntax: str,
     output_file_path: Optional[str] = None,
     background_color: Optional[str] = "white",
+    file_type: Optional[str] = "png",
```

Comment:
could you make the type `Optional[Literal["jpeg", "png", "webp"]]`

Reply:
Thank you for the review.
`typing.Literal` is imported and type annotation is updated as suggested.